src/renderer/modules/common/hljs.ts

```diff
@@ -1,4 +1,4 @@
-import type HighlightJS from "highlightjs";
+import type HighlightJS from "highlight.js";
```

Comment:
Why? Do we really need to bring in the full hljs library as a dependency, even though we're just using the types? That's the whole point of the types package.

Reply:
I did it because I noticed that it is a package that has been deprecated for several years, remaining at v9 of hljs, because from v10 they brought all the types to the main package. Discord uses v11 of hljs and therefore the types were incorrect with the v9 types.